src/routes/anime/+layout.svelte

```diff
@@ -137,200 +137,209 @@
+									is_active
+										? 'relative !bg-accent before:absolute before:-left-[0.15vw] before:z-10 before:h-[1.25vw] before:w-[0.25vw] before:rounded-full before:bg-primary'
+										: '!bg-transparent'
+								)} "
+							>
+								<div class="inline-grid">
+									{#if is_active}
+										<div
+											transition:blur
+											class="absolute inset-0 flex flex-col items-center justify-center gap-[0.5vw]"
+										>
+											<icon class="!text-secondary">
+												<svelte:component this={item_icon.component} class={item_icon.class} />
+											</icon>
 										</div>
-									</a>
-								{/each}
-							{/snippet}
-						</VercelHover>
-					</div>
-					<div class="flex flex-col items-center">
-						{#each Object.entries(icon_mapping.bottom) as item}
-							{@const item_icon = item[1].icon}
-							{@const item_name = item[0]}
+									{:else}
+										<div
+											transition:blur
+											class="absolute inset-0 flex flex-col items-center justify-center gap-[0.5vw]"
+										>
+											<icon class={cn("!text-white")}>
+												<svelte:component this={item_icon.component} class={item_icon.class} />
+											</icon>
 
-							<button
-								type="button"
-								class="btn h-[4vw] w-[4vw] flex-col justify-center gap-[0.45vw] border-none !bg-transparent p-0 text-sm"
-							>
-								<svelte:component this={item_icon.component} class={item_icon.class} />
-								<span class="!m-0 text-[0.75vw] font-semibold capitalize leading-[1.05vw]">
-									{$t(`home.sidebar.${item_name}`)}
-								</span>
-							</button>
+											<span class="text-[0.75vw] font-semibold capitalize leading-[1.05vw]">
+												{$t(`home.sidebar.${item_name}`)}
+											</span>
+										</div>
+									{/if}
+								</div>
+							</a>
 						{/each}
-					</div>
-				</div>
-			</aside>
-			<div id="page" class="flex flex-1 flex-col">
+					{/snippet}
+				</VercelHover>
+			</div>
+			<div class="flex flex-col items-center">
+				{#each Object.entries(icon_mapping.bottom) as item}
+					{@const item_icon = item[1].icon}
+					{@const item_name = item[0]}
+
+					<button
+						type="button"
+						class="btn h-[4vw] w-[4vw] flex-col justify-center gap-[0.45vw] border-none !bg-transparent p-0 text-sm"
+					>
+						<svelte:component this={item_icon.component} class={item_icon.class} />
+						<span class="!m-0 text-[0.75vw] font-semibold capitalize leading-[1.05vw]">
+							{$t(`home.sidebar.${item_name}`)}
+						</span>
+					</button>
+				{/each}
+			</div>
+		</aside>
+		<main class="relative w-full overflow-y-scroll">
+			<svg
+				class="absolute left-0 top-0 z-10 md:size-[2vw]"
+				viewBox="0 0 24 24"
+				fill="none"
+				xmlns="http://www.w3.org/2000/svg"
+			>
+				<path
+					class="fill-secondary"
+					fill-rule="evenodd"
+					clip-rule="evenodd"
+					d="M24 0H0V24C0 10.7451 10.7452 0 24 0Z"
+				/>
+			</svg>
```

Comment:
what is this?

Reply:
a graceful design concept
 ~ by me and titan